app/models/bet_recommendation.rb

```
@@ -0,0 +1,85 @@
+    when 'away'
+      if bet_type == 'spread'
+        "#{game.away_team_name} #{game_odd.formatted_away_line}"
+      else
+        "#{game.away_team_name} #{game_odd.formatted_away_moneyline}"
+      end
+    when 'over'
+      "Over #{game_odd.total_points}"
+    when 'under'
+      "Under #{game_odd.total_points}"
+    end
+  end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `recommendation` is too high. [<0, 24, 11> 26.4/22]

(Metrics/AbcSize)